### PR TITLE
[EMCAL-903] Update gamma/jet trigger and EMCAL observables in jet finder

### PR DESCRIPTION
**Pull request by mfasDa**
- Jet finder: Add cuts on
  - Cluster energy
  - Cluster time
  - Cluster exoticity (default: true)
- Trigger task
  - Add new tags for
    - EMCAL readout
    - Neutral jet trigger
    - Low threshold gamma triggers
- Gamma trigger:
  - Low and high thresholds
  - Cluster and exoticity cut
  - Setter for observable type pt or energy
- Jet trigger
  - Settable jet type (full or neutral jet)
- QA task:
  - Add status bits for new trigger flags
  - Add histograms for low threshold for gamma trigger
  - Settable observable type for clusters (energy or pt)

### CODE REVIEW

**Comment by mfasDa** [comment]
Cluster time and exoticity cut are absolutely crucial, both for the jet finder and the gamma trigger.

**Review by mpuccio — APPROVED** [review]

**Comment by mfasDa** [comment]
Please do not yet merge! I am still testing and debugging. There will be at least one more iteration.

**Review by nzardosh — APPROVED** [review]

**Comment by mfasDa** [comment]
Changes are tested, PR is ready for merging.

**Review by nzardosh — APPROVED** [review]

**Comment by mfasDa** [comment]
@mpuccio Since I pushed some modifications after your approval I need again your approval before it can get merged.

**Review by mpuccio — APPROVED** [review]

**Pull request merged**
